plugin/src/main/java/io/jenkins/plugins/casc/core/JenkinsConfigurator.java

```diff
@@ -56,11 +55,22 @@ protected Jenkins instance(Mapping mapping, ConfigurationContext context) {
                 .setter( noop() ));
 
         // Override "nodes" getter so we don't export Nodes registered by Cloud plugins
-        Attribute.<Jenkins,List<Node>>get(attributes, "nodes").ifPresent(a ->
-            a.getter(j -> j.getNodes().stream()
-                    .filter(node -> node.getDescriptor().isInstantiable())
-                    .collect(Collectors.toList())
-            )
+        Attribute.<Jenkins, List<Node>>get(attributes, "nodes").ifPresent(attribute ->
+                attribute
+                        .getter(jenkins -> jenkins.getNodes().stream()
+                                .filter(node -> !isCloudNode(node))
+                                .collect(Collectors.toList()))
+                        .setter((jenkins, configuredNodes) -> {
```

Comment:
Regarding my comment on https://github.com/jenkinsci/configuration-as-code-plugin/issues/715#issuecomment-476883521 This looks like I was right, we were missing a setter :smile:

Reply:
:)